.travis.yml

```diff
@@ -11,19 +12,35 @@ jobs:
         CXX: clang++
         CC: clang
 
+    #### MAC OS #######
     - os: osx
       osx_image: xcode10.1
       language: generic
       env:
         CXX: clang++
         CC: clang
+
+    #### WINDOWS #######
```

Comment:
This header needs to be lower; the following stanza is still OSX.

Are the headers necessary, I usually find these sorts of things are prone to error.

Reply:
True, I will remove them.